pkg/kubelet/nodeshutdown/nodeshutdown_manager.go

```diff
@@ -83,6 +83,10 @@ func (managerStub) ShutdownStatus() error {
 }
 
 const (
+	// kubelet admission failure reason.
+	NodeShutdownNotAdmittedReason  = "NodeShutdown"
+	nodeShutdownNotAdmittedMessage = "Pod was rejected as the node is shutting down."
```

Comment:
```
ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/pkg/kubelet/nodeshutdown/nodeshutdown_manager_windows.go:56:2: nodeShutdownNotAdmittedMessage redeclared in this block
ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/pkg/kubelet/nodeshutdown/nodeshutdown_manager.go:88:2: 	other declaration of nodeShutdownNotAdmittedMessage
```
The CI needs to be fixed.
/lgtm cancel

Reply:
Fixed. Looks like we added support for graceful node shutdown in windows today and my PR broke windows build